src/engraving/rw/ireader.h

```diff
@@ -60,7 +60,8 @@ class IReader
 public:
     virtual ~IReader() = default;
 
-    virtual Err readScore(Score* score, XmlReader& xml, rw::ReadInOutData* out) = 0;
+    virtual Err readScore(Score* score, XmlReader& xml, rw::ReadInOutData* out,
+                          std::optional<double> spatium = std::nullopt) = 0;
```

Comment:
It would be better to add it to ReadInOutData instead

Reply:
But I can't pass ReadInOutData to loadMscz

Reply:
I wanted to write about the same thing, ReadInOutData is specifically needed for input and output parameters, we can add this spatium there and rename the `out` to `inout` variable